### PR TITLE
fix resource path for windows ;

### PR DESCRIPTION
**Pull request by hoshiryu**
UPDATE the form below to describe your PR.

* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

Be aware that the PR request cannot be accepted if it doesn't pass the Continuous Integration tests.

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Comment by nmellado** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- src/Core/Resources/Resources.cpp  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/STORM/Radium-Engine/pullRequest?prid=4957604&bid=16382450)

**Comment by nmellado** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- src/Core/Resources/Resources.cpp  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/STORM/Radium-Engine/pullRequest?prid=4957604&bid=16382450)

**Comment by nmellado** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- src/Core/Resources/Resources.cpp  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/STORM/Radium-Engine/pullRequest?prid=4957604&bid=16382450)

File: src/Core/Resources/Resources.cpp [hunk 1]
```diff
@@ -28,6 +31,7 @@ std::string getBaseDir() {
 
 ResourcesLocator::ResourcesLocator( void* symbol, const std::string& offset ) : m_basePath{""} {
     auto libraryPath = cpplocate::getLibraryPath( symbol );
+    std::replace(libraryPath.begin(), libraryPath.end(), '\\', '/');
```
Comment (dlyr) [inline src/Core/Resources/Resources.cpp]: maybe use an helper function to have something like
auto libraryPath = getLibraryPath( getRadiumResourcesDir);
in both function

Comment (MathiasPaulin) [reply src/Core/Resources/Resources.cpp]: I agree with @dlyr

**Comment by nmellado** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- src/Core/Resources/Resources.cpp  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/STORM/Radium-Engine/pullRequest?prid=4957604&bid=16382450)

File: src/Core/Resources/Resources.cpp (hunk 1 above)
Comment (MathiasPaulin) [reply src/Core/Resources/Resources.cpp]: This locator code does not work when running on the builtree from XCode. 
see commit fcb1de2a130e01ae1834a67db7b7965a2ca17049 on PR #476 to fix it (also remove duplicated code)

**Comment by nmellado** [comment]
@hoshiryu could you please update your PR according to the requested changes ?

**Comment by MathiasPaulin** [comment]
> @hoshiryu could you please update your PR according to the requested changes ?

@hoshiru, the commit fcb1de2a130e01ae1834a67db7b7965a2ca17049 from the PR #476 fixe some of the required changes.

**Pull request merged**